src/wiki/useful-windows-tips.md

```diff
@@ -269,12 +269,12 @@ Every so often you'd find yourself in a pinch when the C drive runs low on space
 
    ![Hiberfile 40%](./img/useful-windows-tips/pwrcfg40.png)
 
-   If you do not wish to use hibernation, but still want to use Fast Startup (e.g. you have an HDD as your boot drive) then use `powercfg /h /type reduced` instead, which will set the hiberfile size to 20% of its original size (see the difference). Note that this disables hibernation.
+   If you do not wish to use hibernation, but still want to use Fast Startup (e.g. you have an HDD as your boot drive) then use <code>powercfg /h /type reduced</code> instead, which will set the hiberfile size to 20% of its original size (see the difference). Note that this disables hibernation.
 
    ![Hiberfile reduced](./img/useful-windows-tips/pwrcfgredux.png)
 
    If you don't need Fast Startup either, use `powercfg /h off` to completely disable hibernation, fast startup, and delete the hibernation file.
-
+   
```

Comment:
As discussed internally.
```suggestion

```

Reply:
I haven't noticed the empty space. Maybe i should always sync commits from my different devices next time.